ui/StatusQ/src/StatusQ/Popups/StatusMenuItem.qml

```diff
@@ -10,7 +10,6 @@ import StatusQ.Popups 0.1
 MenuItem {
     id: root
 
-    implicitWidth: parent ? parent.width : 0
```

Comment:
aka "too much code"

Reply:
Actually, on a second thought, we might want to set some absolute max value, so that the menu width doesn't explode with overly long strings

Reply:
I was thinking on that and didn't add a max value because I think elided menu items are just bugs, so the right fix IMHO would be to change the phrasing to something appropriate for menu items.

But I'm happy to do it if you think it's better to have it.

Reply:
Yeah that makes sense except for when the content gets generated, eg a wallet address?

Reply:
I guess a default of 640 would do? I've updated the PR.